### PR TITLE
Change the `extended library` word to `Ballerina library`

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Change the `extended library` word to `Ballerina library`.

Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Try out Ballerina's development capabilities using the [Ballerina extension for
 >**Tip:** If you are unsure whether you have found a bug, search the existing issues in the GitHub repo and raise it in the [Ballerina Discord](https://discord.com/invite/wAJYFbMrG2) or [Stack Overflow](https://stackoverflow.com/questions/tagged/ballerina).
 
   - Language, Tooling, Ballerina library, Website: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo
-  - Extended library: <a href="https://github.com/ballerina-platform/ballerina-extended-library/issues">ballerina-extended-library</a> repo
+  - Ballerina library: <a href="https://github.com/ballerina-platform/ballerina-extended-library/issues">ballerina-extended-library</a> repo
```
Comment (keizer619) [inline README.md:66]: Ballerina lib issues are here
https://github.com/ballerina-platform/ballerina-standard-library

Check with team are maintaining both repos or same

File: misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_2.bal [hunk 2]
```diff
@@ -7,7 +7,7 @@ import ballerina/io as console;
 import ballerina/lang.'int;
 import ballerina/log as logger;
 import ballerina/math;
-// Ballerina extended library imports
+// Ballerina library imports
```
Comment (anupama-pathirage) [inline misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_2.bal:10]: Incorrect to have this import here as above lines (7,8,9) are also library imports.

File: misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_3.bal [hunk 3]
```diff
@@ -8,7 +8,7 @@ import ballerina/io as console;
 import ballerina/lang.'int;
 import ballerina/log as logger;
 import ballerina/math;
-// Ballerina extended library imports
+// Ballerina library imports
```
Comment (anupama-pathirage) [inline misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_3.bal:11]: ditto

File: misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_3.bal [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
 import greeter.util;
-// Third-Party library imports
-import apache/kafka as apache_kafka;
+
+// Ballerina library imports
 import ballerina/io as console;
 // Ballerina lang library import of int
 // Second line of comments
-import ballerina/lang.'int;
+import ballerina/lang.'int;
```
Comment (keizer619) [inline misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_3.bal:8]: ``
```suggestion
import ballerina/lang.'int;
```

File: misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_2.bal (hunk 2 above)
Comment (praneesha) [reply misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_2.bal]: 1. Corrected.

File: README.md (hunk 1 above)
Comment (praneesha) [reply README.md]: Corrected.

File: misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_3.bal (hunk 3 above)
Comment (praneesha) [reply misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_3.bal]: Corrected.

File: README.md (hunk 1 above)
Comment (praneesha) [reply README.md]: > Corrected. Need to wait for the repo rename and URL redirection.

Comment (praneesha) [reply README.md]: The repo is renamed and the URL is redirected now.

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**